### ISSUE TITLE
Deduplication of react

### ISSUE THREAD

**Issue by evepen1** [raisezhang/react-drag-listview#13]
After installing react-drag-listview, I'm facing a deduped react package and the application is crashing. peer dependencies should be declared for react.

I'm getting the following error: 

` Uncaught Error: Element ref was specified as a string (dragList) but no owner was set. This could happen for one of the following reasons:
1. You may be adding a ref to a functional component
2. You may be adding a ref to a component that was not created inside a component's render method
3. You have multiple copies of React loaded`

**Comment by raisezhang** [comment]
Thank you for your feedback. I'll look at this question tomorrow and give you an answer.

**Comment by raisezhang** [comment]
Hey  @evepen1 , what's u React version installed ? 

I think this problem is caused by the inconsistency of the React version. The React version used by the 'react-drag-listview'  is ^16.0.0.

If they are consistent, try to modify the resolve path of React . 

Hope that can helped u~    Thanks    

ref： 
https://github.com/facebook/react/issues/12921 
https://github.com/JedWatson/react-select/issues/2025

**Comment by raisezhang** [comment]
Hey @evepen1 ,   What's u React version in project？ 

I think this problem is caused by the inconsistency of the React version. The React version used by the react-drag-listview is ^16.0.0.

If they are consistent, try to modify the resolve path of React? 

Hope that helped u~   Thanks 

ref:
https://github.com/JedWatson/react-select/issues/2025
https://github.com/facebook/react/issues/12626

**Comment by evepen1** [comment]
Hi Zhang,

I'm using React 16.3.xx version. So should I switch back to 16.0.0 to
resolve the issue ?

Thanks for your help again.

**Comment by raisezhang** [comment]
@evepen1    

You can add peerDependencies to package.json in your project to solve this problem.

```
"peerDependencies": {
    "react": ">=16.0.0",
    "react-dom": ">=16.0.0"
 }
```

**Comment by evepen1** [comment]
@raisezhang 

I have added my react & react-dom latest versions as peerDependencies but still getting the same error as above.

**Comment by raisezhang** [comment]
https://unpkg.com/react-drag-listview@0.0.9/package.json   
https://unpkg.com/react-drag-listview@0.1.3/package.json

Which version are you using?  Recommended to use the latest version！    

```
"react-drag-listview": "^0.1.3"
```

**Comment by evepen1** [comment]
The issue got resolved and `"react-drag-listview": "^0.1.3"` 
is working fine..!

Thanks Zhang
